Mathlib/Topology/Algebra/Module/Alternating.lean

```diff
@@ -0,0 +1,616 @@
+/-
+Copyright (c) 2023 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov, Heather Macbeth, Sébastien Gouëzel
+-/
+import Mathlib.LinearAlgebra.Alternating
+import Mathlib.Topology.Algebra.Module.Multilinear
+
+/-!
+# Continuous alternating multilinear maps
+
+In this file we define bundled continuous alternating maps and develop basic API about these
+maps, by reusing API about continuous multilinear maps and alternating maps.
+
+## Keywords
+
+multilinear map, alternating map, continuous
+-/
+
+open Function Matrix
+
+open scoped BigOperators
+
+/-- A continuous alternating map is a continuous map from `ι → M` to `N` that is
+
+- multilinear : `f (update m i (c • x)) = c • f (update m i x)` and
+  `f (update m i (x + y)) = f (update m i x) + f (update m i y)`;
+- alternating `f v = 0` whenever `v` has two equal coordinates.
+-/
+structure ContinuousAlternatingMap (R M N ι : Type _) [Semiring R] [AddCommMonoid M] [Module R M]
+    [TopologicalSpace M] [AddCommMonoid N] [Module R N] [TopologicalSpace N] extends
+    ContinuousMultilinearMap R (fun _ : ι => M) N where
+  map_eq_zero_of_eq' : ∀ (v : ι → M) (i j : ι), v i = v j → i ≠ j → toFun v = 0
```

Comment:
can't you extend `AlternatingMap R M N ι` instead of putting this explicit condition?

Reply:
It was forward-ported from Lean 3. I'll try later today.